Godot_test/ClientScripts/GameEventHandler.cs

```diff
@@ -83,18 +95,52 @@ public void ActionsManager_ActionAppended(object sender, GameEvent ev)
             }
             else if (ita.InteractiveKind == InteractiveActionKind.Destroyed)
             {
-              var barrel = GameLevel.interactiveList[(Barrel)ita.InvolvedTile];
-              var anim = (AnimationPlayer)barrel.GetNode("AnimationPlayer");
-              anim.Play("destroy");
+              var barrel = GameLevel.interactiveList[ita.InvolvedTile];
```

Comment:
Make interactiveList and lootingList private mebers of GameLevel. Add there method:
 Node GetNode(Tile tile) and inside search for a Node in Dictionaries.